__tests__/integration.test.js

```diff
@@ -103,15 +103,38 @@ describe("getArticles", () => {
         });
       });
   });
-  it("should return the object in the correct order, with correct length and order", () => {
+  it("should return status 200 plus the object in the correct order, with correct length and order", () => {
     return request(app)
       .get("/api/articles")
+      .expect(200)
       .then((res) => {
         const { articles } = res.body;
         expect(articles).toHaveLength(13);
         expect(articles).toBeSortedBy("created_at", { descending: true });
       });
   });
+  it('should return correct object when passed "topic" as a query and status 200', () => {
+    return request(app)
+    .get("/api/articles?topic=mitch")
+    .expect(200)
+    .then((res) => {
+      const { articles } = res.body;
+      expect(articles).toHaveLength(12);
+      articles.forEach((article) => {
+        expect(article.topic).toBe("mitch");
+      });
+    })
+  });
+  it('should return status 200 plus empty array when topic does not exist', () => {
+    return request(app)
+    .get("/api/articles?topic=dogs")
```

Comment:
This would be a 404 
A 200 would happen when a topic exists in the database but has no article associated, like for the topic paper 🙂